Libraries/Components/WebView/WebView.android.js

```diff
@@ -195,6 +198,16 @@ var WebView = React.createClass({
       console.warn('WebView: `source.body` is not supported when using GET.');
     }
 
+    var onShouldOverrideUrlLoading = this.props.onShouldStartLoadWithRequest && ((event: Event) => {
```

Comment:
just some formal rants (aka nits): The formatting here makes this particularly hard to read. Please add some line breaks, and white space. Thank you.